src/app/components/change-password/change-password.component.ts

```diff
@@ -80,10 +81,14 @@ export class ChangePasswordComponent implements OnInit {
             this.errorMessage = ''
             this.changePassServ.sendPasswordData(username, newPassword).subscribe(
               (data) => {
-                this.changePasswordSuccess = 1
+                if(data == true){
```

Comment:
@mbaileyfuturist Please remove unnecessary boolean logic.  The Sonar Cloud report on this code smell can be found [here ](https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=7&id=Revature-CashOverflow-2_CashOverflow-angular&open=AYALy1b1az4AdU3QB0qY)👈 

Boolean literals should be avoided in comparison expressions == and != to improve code readability.

Change to `if (data) { ... }`

Reply:
Ok redundant boolean logic has been removed.